packages/size-limit/run.js

```diff
@@ -43,10 +50,20 @@ module.exports = async process => {
 
     config = await getConfig(plugins, process, args, pkg)
 
-    await calc(plugins, config, ora)
+    if (hasArg('--watch')) {
```

Comment:
We need to call `mainCalc` first before starting to watch.

Reply:
Yeah, you're right! Fixed